lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -263,8 +290,13 @@ def get_d3_sequential_open_distrib(course_id):
             subsection_name = own_metadata(subsection).get('display_name', '')
 
             num_students = 0
-            if subsection.location in sequential_open_distrib:
-                num_students = sequential_open_distrib[subsection.location]
+
+            try:
+                module = client.modules(course_id, subsection.location)
```

Comment:
Is it possible to make one api call with a list of modules? This seems a bit inefficient if so.